influence_benchmark/stats/preferences_per_iteration.py

```diff
@@ -16,17 +16,30 @@ def load_trajectories(trajectory_path: Path) -> pd.DataFrame:
 
 
 def get_top_n_trajectories(trajectory_path: Path, num_chosen_trajectories: int) -> List[Dict]:
+    return get_func_n_trajectories(trajectory_path, num_chosen_trajectories, pd.DataFrame.nlargest)
```

Comment:
We can probably remove this and always use the best_worst method. Speed gains are probably miniscule

Reply:
I think removing it makes some other code less nice to read?